### PR TITLE
Adding Kameleoon (Actions) Destination documentation

### PR DESCRIPTION
**Pull request by yhassan-kam**
### Proposed changes

I created the doc file for the new Kameleoon (Actions) Destination. This integration is currently in private beta.

### Merge timing
ASAP once approved
(We are currently testing the integration with one customer, and we are looking forward to being listed in the Catalog, switch from private beta to beta.)

### CODE REVIEW

File: src/connections/destinations/catalog/actions-kameleoon/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+     - **API Key**: Paste your Kameleoon API key. To generate an API Key, see [Kameleoon's documentation](https://help.kameleoon.com/){:target="_blank”}.
+     - **Sitecode**: Paste your Kameleoon project sitecode. You can find it in the [project dashboard](https://help.kameleoon.com/question/how-do-i-find-my-site-id/){:target="_blank”}.
+6. Enable the destination by clicking the **Enable Destination** toggle switch.
+7. Click **Save Changes**.
+
+
+{% include components/actions-fields.html %}
+
+
+The integration requires that you use the same system of identifiers for both tools. While Segment uses the userId, Kameleoon uses the kameleoonVisitorCode. In order to identify which visitor triggered the forwarded Segment events, you must include the kameleoonVisitorCode inside your Segment events. To know more, see [Kameleoon's documentation page](https://help.kameleoon.com/){:target="_blank”}.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-kameleoon/index.md:43]: @yhassan-kam can you provide the exact page link for the kameleoon documentation page?

File: src/connections/destinations/catalog/actions-kameleoon/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+## Migration from the classic Kameleoon destination
+
+To migrate from the classic Kameleoon destination:
+1. Include the `kameleoonVisitorCode` in your Segment events for accurate visitor tracking. To know more, see [the dedicated documentation page](https://help.kameleoon.com/){:target="_blank”}.
```
[the remark this answers is not in the document]
Comment (yhassan-kam) [reply src/connections/destinations/catalog/actions-kameleoon/index.md]: @stayseesong I added the link in the last commit. 
[https://help.kameleoon.com/setting-up-segment-destination-actions/#Matching_an_Segmentio_user_with_a_Kameleoon_visitor](https://help.kameleoon.com/setting-up-segment-destination-actions/#Matching_an_Segmentio_user_with_a_Kameleoon_visitor)

File: src/connections/destinations/catalog/actions-kameleoon/index.md (hunk 1 above)
Comment (yhassan-kam) [reply src/connections/destinations/catalog/actions-kameleoon/index.md]: @stayseesong I added the link in the last commit. 
[https://help.kameleoon.com/setting-up-segment-destination-actions/#Matching_an_Segmentio_user_with_a_Kameleoon_visitor](https://help.kameleoon.com/setting-up-segment-destination-actions/#Matching_an_Segmentio_user_with_a_Kameleoon_visitor)

File: src/connections/destinations/catalog/actions-kameleoon/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Getting started
+
+1. Navigate to **Connections > Catalog** in the Segment web app.
+2. Search for *Kameleoon (Actions)* and select the destination.
+3. Click **Add destination**.
+4. Select the Source you want to connect to Kameleoon (Actions) and click **Confirm Source**.
+5. On the **Basic Settings** side panel, complete the required fields:
+     - **Name**: Enter a name to help you identify this destination in Segment
+     - **API Key**: Paste your Kameleoon API key. To generate an API Key, see [Kameleoon's documentation](https://help.kameleoon.com/){:target="_blank”}.
```
[the remark this answers is not in the document]
Comment (yhassan-kam) [reply src/connections/destinations/catalog/actions-kameleoon/index.md]: @stayseesong I added the link in the last commit. 
[https://help.kameleoon.com/setting-up-segment-destination-actions/#Kameleoon_setup](https://help.kameleoon.com/setting-up-segment-destination-actions/#Kameleoon_setup)

**Comment by stayseesong** [comment]
@tcgilbert do you have the destination id for this?

**Review by tcgilbert — APPROVED** [review]

**Pull request merged**